src/stylesheets/components/navbar.scss

```diff
@@ -0,0 +1,97 @@
+@keyframes fadein {
+  0% {
+    opacity: 0%;
+  }
+  100% {
+    opacity: 100%;
+  }
+}
+
+.navbar {
+  min-height: 100vh;
+  width: $navbar-width;
+  background: $solid-background;
+  box-shadow: 0 0 4px 5px rgba($color: black, $alpha: 0.25);
+  padding: 30px 0px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  gap: 30px;
```

Comment:
there isn't much support for using `gap` with flexboxes, it's mainly support for grid layouts. Can you change this to margins?